Foundation/NSConcreteValue.swift

```diff
@@ -11,7 +11,7 @@ import CoreFoundation
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || IS_CYGWIN
```

Comment:
If we do what I said above, would os(Linux) just work for cygwin? I could understanding needing the platform difference for CF (which has a lot of platform specific stuff in it), but I'm not sure we need to distinguish between the two from the Swift code.